project-base/app/assets/js/admin/grapesjs/plugins/grapesjs-custom-iframe-plugin.js

```diff
@@ -17,22 +17,29 @@ export default grapesjs.plugins.add('custom-iframe', (editor) => {
                         type: 'text',
                         label: 'Width',
                         name: 'width',
-                        value: '100%'
+                        placeholder: '100%'
                     },
                     {
                         type: 'text',
                         label: 'Height',
                         name: 'height'
                     }
                 ]
+            },
+            init () {
+                this.on('change:attributes:type', this.handleTypeChange);
+            },
+
+            handleTypeChange () {
+                console.log('Input type changed to: ', this.getAttributes().type);
```

Comment:
I do not think this would do any harm, but I need to ask - do we want to keep this (and use it on production) or was it just a debugging thing that should be removed now? If I am not mistaken, generally by standards, we do not allow the `console.log` usage

Reply:
Ah, leftover after debugging. Removed.